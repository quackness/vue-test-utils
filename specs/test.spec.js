import TestComponent from '@/test'
import List from '@/test'
import { mount, shallowMount } from '@vue/test-utils'


// test('first spec', () => {
//   console.log(TestComponent)
// })
// test('mount a vue comp', () => {
//   const wrapper = mount(TestComponent, {
//     propsData: {
//       value: 'VueS'

//     }
//   })
//   console.log(wrapper)
//expect(wrapper.html()).toMatchSnapshot()
//})

// test('ListComponent shallow', () => {
//   console.log(mount(List).html())
//   console.log(shallowMount(List).html())
// })

test('List component', () => {
  const wrapper = mount(List)
  const movies = wrapper.vm.marvelMovies || ['']


  wrapper.setData({ marvelMovies: [...movies, 'Endgame'] })

  expect(wrapper.html()).toMatchSnapshot()
})