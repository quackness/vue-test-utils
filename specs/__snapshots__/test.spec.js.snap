// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount a vue comp 1`] = `
<div>
  <h1>Hello Test</h1>
</div>
`;
